wiki/Client/Keyboard_shortcuts/de.md

```diff
@@ -1,253 +1,295 @@
-- `Strg` + `Shift` + `L`: Änderungen zurücksetzen (vollständig): Lädt zusätzlich noch Hitsounds und Skinelemente neu
-- `Strg` + `Shift` + `O`: Die Difficulty-Auswahl öffnen
-- `Strg` + `S`: Speichern
-- `Strg` + `N`: Alle Objekte löschen (aber Timing und Song Setup behalten)
-- `Strg` + `Pfeil unten`: Wiedergabegeschwindigkeit um 25% verringern
-- `Strg` + `Shift` + `Pfeil unten`: Wiedergabegeschwindigkeit um 5% verringern
-- `Strg` + `Up`: Wiedergabegeschwindigkeit um 25% erhöhen
-- `Strg` + `Shift` + `Pfeil oben`: Wiedergabegeschwindigkeit um 5% erhöhen
-- `F1`: Zum [Compose](/wiki/Client/Beatmap_editor/Compose)-Modus wechseln
-- `F2`: Zum [Design](/wiki/Client/Beatmap_editor/Design)-Modus wechseln
-- `F3`: Zum [Timing](/wiki/Client/Beatmap_editor/Timing)-Modus wechseln
-- `F4`: Das [Song Setup](/wiki/Client/Beatmap_editor/Song_Setup) öffnen (neues Fenster)
-- `F5`: Den Testmodus starten
-- `F6`: Das Timing Setup öffnen
+Diese Kürzel funktionieren überall im Beatmap-Editor:
+
+| Kürzel | Aktion |
+| :-- | :-- |
+| `Z` | Springe zur ersten Note oder zum Anfang des Songs. |
+| `V` | Springe zur letzten Note oder zum Ende des Songs. |
+| `X` | Spiele den Song von Beginn an. |
+| `C` oder `Leertaste` | Pausieren oder Wiedergabe fortsetzen. |
+| `Pfeil links` oder `Pfeil rechts` | Durch den Song springen. |
+| `Pfeil oben` or `Pfeil unten` | Springe zwischen Timing Points. |
+| `Shift` + `Pfeil links` oder `Shift` + `Pfeil rechts` | Springe durch den Song mit 4-facher Geschwindigkeit. |
+| `Strg` + `B` | Setze ein Lesezeichen an der aktuellen Stelle. |
+| `Strg` + `Shift` + `B` | Entferne das Lesezeichen an der aktuellen Stelle. |
+| `Strg` + `Pfeil links/rechts` | Springe zum vorherigen/nächsten Lesezeichen. |
+| `Strg` + `L` | Setze die Änderungen zurück (teilweise). Das wird die `.osu`- und `.osb`-Dateien neu laden. |
+| `Strg` + `Shift` + `L` | Setze die Änderungen zurück (vollständig). Das wird alles neu laden. |
+| `Strg` + `O` | Öffne das Menü zur Auswahl der Schwierigkeitsstufe. |
+| `Strg` + `S` | Speichert die Beatmap in ihrem aktuellen Zustand. |
+| `Strg` + `N` | Lösche alle Hit-Objekte. |
+| `Strg` + `Pfeil hoch/runter` | Erhöhe/verringere die Wiedergabegeschwindigkeit um 25 %. |
+| `Strg` + `Shift` + `Pfeil hoch/runter` | Erhöhe/verringere die Wiedergabegeschwindigkeit um 5 %. |
+| `F1` | Wechsle zum [Compose](/wiki/Client/Beatmap_editor/Compose)-Modus. |
+| `F2` | Wechsle zum [Design](/wiki/Client/Beatmap_editor/Design)-Modus. |
+| `F3` | Wechsle zum [Timing](/wiki/Client/Beatmap_editor/Timing)-Modus. |
+| `F4` | Öffne den Dialog für das [Song-Setup](/wiki/Client/Beatmap_editor/Song_Setup). |
+| `F5` | Starte den Testmodus. |
+| `F6` | Öffne den Dialog für das Timing und die Control Points. |
+| `Strg` + `P` | Add an uninherited timing point at the current timestamp. If not in the timing mode, this will also open the timing and control points dialog. |
```

Comment:
```suggestion
| `Strg` + `P` | Füge einen unvererbten Timing Point an der aktuellen Stelle hinzu. Wenn dies nicht im Timing-Mode geschiet, wird auch der Dialog für das Timing und die Control Points geöffnet |
```

Reply:
Really sorry that I've forgot to translate this and the other one further above 😓

Reply:
That's what reviews are for i guess xd